app/components/app_patient_details_component.rb

```diff
@@ -15,24 +15,8 @@ def initialize(session:, patient: nil, consent_form: nil, school: nil)
 
   private
 
-  def known_as
-    if @object.respond_to? :common_name
-      @object.common_name
-    elsif @object.respond_to? :preferred_name
-      @object.preferred_name
-    end
-  end
-
-  def date_of_birth
-    if @object.respond_to? :dob
-      @object.dob
-    else
-      @object.date_of_birth
-    end
-  end
-
   def aged
-    "aged #{date_of_birth ? @object.age : ""}"
+    "aged #{@object.date_of_birth ? @object.age : ""}"
```

Comment:
This method will produce `(aged )` if we don't have a DOB for some reason (is that even possible?). Is that definitely what's desired?

Reply:
Might not be possible but that's worth fixing anyway. There's no good situation where we'd want `(aged )`

Reply:
Good spot, maybe it should just be `@object.date_of_birth` as it should always be present

Reply:
We've decided to hide the `Date of birth` field here altogether if the `date_of_birth` is missing from the patient.

Reply:
👍